sormas-backend/src/main/java/de/symeda/sormas/backend/info/EntityColumn.java

```diff
@@ -49,7 +51,7 @@ public enum EntityColumn {
 	DATA_PROTECTION(256 * 30, EntityColumn::getDataProtection, false, true, false),
 	CAPTION(256 * 30, EntityColumn::getCaption, false, true, false),
 	DESCRIPTION(256 * 60, EntityColumn::getDescription, true, true, false),
-	REQUIRED(256 * 10, EntityColumn::getRequired, false, true, false),
```

Comment:
Let's keep the enum name "REQUIRED" otherwise translation will have to be updated.